pallets/validators-manager/src/lib.rs

```diff
@@ -396,270 +417,522 @@ pub use default_weights::WeightInfo;
 
 pub const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
 
-// TODO [TYPE: review][PRI: medium]: if needed, make this the default value to a configurable
-// option. See MinimumValidatorCount in Staking pallet as a reference
-const DEFAULT_MINIMUM_VALIDATORS_COUNT: usize = 2;
-
 pub type AVN<T> = avn::Pallet<T>;
 
 impl<T: Config> Pallet<T> {
-    fn start_activation_for_registered_validator(
-        registered_validator: &T::AccountId,
-        tx_id: EthereumId,
-    ) {
-        let ingress_counter = Self::get_ingress_counter() + 1;
+    /// Helper function to compress eth public key (exposed for testing and benchmarking)
+    #[cfg(any(test, feature = "runtime-benchmarks"))]
+    pub fn compress_eth_public_key(full_public_key: sp_core::H512) -> ecdsa::Public {
+        compress_eth_public_key(full_public_key)
+    }
```

Comment:
why do we need the helper function on the pallet and tests and benchmarks import directly the primitive dependency where needed?